app/screens/riff_video_chat/webview/webview.js

```diff
@@ -46,6 +47,10 @@ export default class RiffWebview extends PureComponent {
             },
         } = this.props;
 
+        if (!riffMeetingBaseUrl) {
+            return false;
```

Comment:
we cannot do this as returning any nullish value like the null or empty string "", react native gives error

Reply:
I think you must be getting an error because you are calling this function for the value of the `uri` attribute of the `source` prop in the `Webview`, otherwise returning null or "" in React native doesn't give an error. For eg - 
https://github.com/mattermost/mattermost-mobile/blob/63ce6c7afbcaf6a4a42f2f66eed37d73e0d85acd/app/components/badge.js#L165